wasm/jit/engine.go

```diff
@@ -251,16 +251,30 @@ type compiledWasmFunction struct {
 	codeInitialAddress uintptr
 	// The same purpose as codeInitialAddress, but for memory.Buffer.
 	memoryAddress uintptr
+	// The max of the stack pointer this function can reach.
```

Comment:
This will help in code navigation a bit as I think what is confusing is that setting this is more like scheduling it to grow vs synchronously doing it.

```suggestion
	// The max of the stack pointer this function can reach. Lazily applied via maybeGrowStack.
```